C/services/north/include/north_service.h

```diff
@@ -57,6 +65,73 @@ class NorthService : public ServiceAuthHandler {
 		bool				getDryRun() { return m_dryRun; };
 		void				alertFailures();
 		void				clearFailures();
+		// Debugger Entry point
```

Comment:
Since the implementations are in the include file, they will be inlined. Is that intended?